runtime/syntax/raku.yaml

```
@@ -1,7 +1,7 @@
 filetype: raku
 
 detect:
-    filename: "(\\.p6$|\\.pl6$|\\.pm6$|\\.raku$|\\.rakumod$|\\.rakudoc$)"
+    filename: "(\\.p(l|m|od)?6$|\\.raku(mod|doc|test)?$|\\.nqp$)"
```

```suggestion
   filename: "\\.(p(l|m|od)?6|raku(mod|doc|test)?|nqp)$"
```

Not fully, since you still kept the `\\.` & `$` per file pattern. Se above, it's shortened once again, as @Andriamanitra mentioned.

---

Argh, there was a white space issue present in the suggestion. :man_facepalming: